library/src/main/java/com/schibsted/spain/barista/util/BitmapComparator.kt

```diff
@@ -0,0 +1,19 @@
+package com.schibsted.spain.barista.util
+
+import android.graphics.Bitmap
+import java.util.Arrays
+
+class BitmapComparator {
+
+  companion object {
+    fun compare(b1: Bitmap, b2: Bitmap) = if (b1.width == b2.width && b1.height == b2.height) {
```

Comment:
I don't know about the agreed Kotlin style, but using = to declare those long methods looks strange to me. I propose using the usual { } when writing methods longer than one line or a simple return.